playbooks/robusta_playbooks/krr.py

```diff
@@ -27,16 +28,18 @@
     ScanType,
     action,
 )
-from robusta.core.model.env_vars import INSTALLATION_NAMESPACE
+from robusta.core.model.env_vars import INSTALLATION_NAMESPACE, RELEASE_NAME, CLUSTER_DOMAIN, load_bool
 from robusta.core.reporting.consts import ScanState
 from robusta.integrations.openshift import IS_OPENSHIFT
 from robusta.integrations.prometheus.utils import generate_prometheus_config
 from robusta.utils.parsing import format_event_templated_string
 
-IMAGE: str = os.getenv("KRR_IMAGE_OVERRIDE", f"{IMAGE_REGISTRY}/krr:v1.23.0")
+IMAGE: str = os.getenv("KRR_IMAGE_OVERRIDE", f"{IMAGE_REGISTRY}/krr:v1.24.0")
 KRR_MEMORY_LIMIT: str = os.getenv("KRR_MEMORY_LIMIT", "2Gi")
 KRR_MEMORY_REQUEST: str = os.getenv("KRR_MEMORY_REQUEST", "2Gi")
 KRR_STRATEGY: str = os.getenv("KRR_STRATEGY", "simple")
+KRR_PUSH_SCAN: str = load_bool("KRR_PUSH_SCAN", True)
```

Comment:
Thinking here loud, @Avi-Robusta @arikalon1 What do you think we should enabled it by default or for the problematic customers ask to enable it and make sure it works?